pokemon-app/src/index.tsx

```diff
@@ -1,16 +1,25 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { applyMiddleware, createStore } from 'redux';
 import App from './App';
+import { allReducers } from './reducers';
 import reportWebVitals from './reportWebVitals';
+import ReduxThunk from 'redux-thunk'
+import { composeWithDevTools } from 'redux-devtools-extension';
+import { Provider } from 'react-redux';
```

Comment:
Please sort imports as described [here](https://dev.to/otamnitram/sorting-your-imports-correctly-in-react-213m)